src/main/scala/api/MyMap.scala

```diff
@@ -0,0 +1,98 @@
+package api
+
+
+import scala.collection.JavaConverters._
+
+import java.util.concurrent.ConcurrentHashMap
+
+import javax.servlet.http.{HttpServlet, HttpServletRequest, HttpServletResponse}
+
+import scala.collection.concurrent
+
+object MyMap {
+  val map: concurrent.Map[String, String] = new ConcurrentHashMap[String, String].asScala
+
+  class getServlet extends HttpServlet {
+    override protected def doGet(req: HttpServletRequest, resp: HttpServletResponse): Unit = {
+      var key = req.getParameter("key")
+      var x = getKey(key)
+      //        println("sleeping" + key)
+      val async = req.startAsync
+      //        Thread.sleep(10000)
+      resp.setContentType("text/html")
+      resp.setStatus(HttpServletResponse.SC_OK)
+      resp.getWriter.println(s"<h2>Get Triggered </h2>" + x)
+      async.complete()
+      return
+    }
+
+    def getKey(key: String): Option[String] = {
+      return map.get(key)
+    }
+  }
+
+  class putServlet extends HttpServlet {
+    override protected def doGet(req: HttpServletRequest, resp: HttpServletResponse): Unit = {
+      var key = req.getParameter("key")
```

Comment:
Can you look at DRYing some of this code? https://en.wikipedia.org/wiki/Don%27t_repeat_yourself